addons/mrp_subcontracting/models/stock_warehouse.py

```diff
@@ -22,7 +22,7 @@ class StockWarehouse(models.Model):
         domain=[('code', '=', 'mrp_operation')])
     subcontracting_resupply_type_id = fields.Many2one(
         'stock.picking.type', 'Subcontracting Resupply Operation Type',
-        domain=[('code', '=', 'outgoing')])
+        domain=[('code', '=', 'internal')])
```

Comment:
Could you write a test to check type code after a warehouse creation (you can check on the default one) + another test on the partner specific location